cmd/deploy-contracts/main.go

```diff
@@ -28,7 +28,7 @@ func main() {
 		&cli.BoolFlag{
 			Name:    START_ANVIL,
 			Usage:   "Specifies whether to start a local anvil instance",
-			Value:   true,
+			Value:   false,
```

Comment:
Can you explain the reasoning for this change?

Reply:
The `anvil` network was starting even when the `CHAIN_URL` was set to Filecoin. It would probably make more sense just to remove the `START_ANVIL` option completely from this script. There are no instances where this script is used with that option set right now. We should just be able to use `CHAIN_URL` to determine where the contracts are deployed, and the user can start `anvil` externally (and keep it running) if thats the network they want to use.

Reply:
Sounds like good cleanup to me, perhaps lets give a chance for folks to object at today's standup and then do that.

Reply:
Removed `anvil` startup from this script altogether: https://github.com/statechannels/go-nitro/pull/1631/commits/913eb225e4e3954ef0b18c99bf00591059e33669